components/input/input.html

```diff
@@ -0,0 +1,99 @@
+            <div class="component-example flex-row flex-align-center flex-justify-center flex-gap1">
+                <div class="component-input">
+                    <input class="input-box" placeholder=" " type="text">
+                    <label class="input-label" for="name">Enter name</label>
+                </div>
+                <div class="component-input">
+                    <input class="input-box" placeholder=" " type="number">
+                    <label class="input-label" for="mobile number">Mobile Number</label>
+                </div>
+            </div>
+            <iframe src="https://carbon.now.sh/embed/QnQbXqSGfIgNB4W5kdwN" class="code-iframe"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+
+            <p class="section-para">Use class "input-box-basic" for normal input box .</p>
+            <div class="component-example flex-row flex-justify-center flex-gap1">
+                <div class="component-input ">
+                    <input class="input-box-basic" placeholder="Enter email" type="text">
+                    <label for="Email"></label>
+                </div>
+                <div class="component-input">
+                    <input class="input-box-basic" placeholder="Enter Data" type="number">
+                    <label for="Data"></label>
+                </div>
+            </div>
+
+            <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=one-dark&wt=none&l=auto&width=714&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522component-input%2520%2522%253E%250A%253Cinput%2520class%253D%2522input-box-basic%2522%2520placeholder%253D%2522Enter%2520email%2522%2520type%253D%2522text%2522%253E%250A%253Clabel%2520for%253D%2522Email%2522%253E%253C%252Flabel%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522component-input%2522%253E%250A%253Cinput%2520class%253D%2522input-box-basic%2522%2520placeholder%253D%2522Enter%2520Data%2522%2520type%253D%2522number%2522%253E%250A%253Clabel%2520for%253D%2522Data%2522%253E%253C%252Flabel%253E%250A%253C%252Fdiv%253E"
+                class="code-iframe" sandbox="allow-scripts allow-same-origin">
+            </iframe>
+            <p class="section-para">Use class "input-box-success" and "input-box-error" for validation input box .</p>
+            <div class="component-example flex-row flex-align-center flex-justify-center flex-gap1">
+                <div class="component-input">
+                    <input class="input-box input-box-success" placeholder=" " type="text">
+                    <label class="input-label" for="name">Success</label>
+                </div>
+                <div class="component-input">
+                    <input class="input-box input-box-error" placeholder=" " type="number">
```

Comment:
You can make Error Input type="text"

Reply:
Just an example to show both text and number input types